README.md

```diff
@@ -15,6 +15,7 @@ None.
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
 | freeipa_days_before_inactive | The number of days a user can go without logging in before his or her account is determined to be inactive and is disabled. | `45` | No |
```

Comment:
Inclusivee language change. The comment in `defaults/main.yml` should also be updated.

```suggestion
| freeipa_days_before_inactive | The number of days a user can go without logging in before their account is determined to be inactive and is disabled. | `45` | No |
```

Reply:
Please see commit 206a290.